app/src/main/res/values/strings.xml

```diff
@@ -313,6 +313,10 @@
     <string name="image_rearranging_options">Image Rearranging Options</string>
     <string name="filter_file_name">%s_%s.jpg</string>
     <string name="reset">Reset</string>
+    <string name="crop">Crop</string>
+    <string name="rotate">Rotate</string>
+    <string name="done">Done</string>
+    <string name="image_successfully_cropped">Image successfully cropped</string>
```

Comment:
add these strings to files of other locales too. This is probably causing the travis to fail.